src/test/java/de/fraunhofer/aisec/cpg/ScopeManagerTest.java

```diff
@@ -0,0 +1,30 @@
+package de.fraunhofer.aisec.cpg;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import de.fraunhofer.aisec.cpg.frontends.LanguageFrontend;
+import de.fraunhofer.aisec.cpg.frontends.TranslationException;
+import de.fraunhofer.aisec.cpg.frontends.java.JavaLanguageFrontend;
+import de.fraunhofer.aisec.cpg.passes.scopes.ScopeManager;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class ScopeManagerTest {
+
+  private TranslationConfiguration config;
+
+  @BeforeEach
+  void setUp() {
+    config = TranslationConfiguration.builder().defaultPasses().build();
+  }
+
+  @Test
+  void testLiteral() throws TranslationException {
```

Comment:
`testEquals`?

Reply:
It does not matter, they don't implement their own equals method. But I can change it.

Reply:
No i meant the wording of the test :)